.github/workflows/ci.yaml

```diff
@@ -38,6 +38,8 @@ jobs:
         with:
           repository: HydrasDB/citus
           token: ${{ secrets.GH_TOKEN }}
+          # Remove this after devel is merged
+          ref: devel
```

Comment:
devel is merged :)

Reply:
Sweet, updated in https://github.com/HydrasDB/hydra/pull/28/commits/5952f91fd428aae6c90b9371c05714b77d729b04